readme.txt

```diff
@@ -126,12 +126,7 @@ If you get stuck, you can ask for help in the Plugin Forum.
 
 == Changelog ==
 
-= 5.5.0 - 2021-09-15 =
```

Comment:
we shouldn't remove this section

Reply:
In the readme we keep only the last release and then a link to the changelog.txt.

Reply:
Oh. I didn't know that. Looks good then!